pkg/api/metrics.go

```diff
@@ -129,10 +129,10 @@ func createMetrics() *Metrics {
 			},
 		),
 		LastRequestUnixNano: time.Now().UnixNano(),
-		QueryBatchDuration: prometheus.NewHistogram(
+		QueryRemoteReadBatchDuration: prometheus.NewHistogram(
 			prometheus.HistogramOpts{
 				Namespace: util.PromNamespace,
-				Name:      "query_batch_duration_seconds",
+				Name:      "query_remote_read_batch_duration_seconds",
 				Help:      "Duration of query batch read calls to the remote storage.",
 				Buckets:   prometheus.DefBuckets,
```

Comment:
```suggestion
				Buckets:   []float64{0.05, 0.1, 0.25, 0.5, 1, 2.5, 5, 10, 30, 50, 100, 250, 500, 1000, 2500},
```
based on what Ramon suggested